### ISSUE TITLE
Unable to detect form submit when the button is not `<input type="submit">`

### ISSUE THREAD

**Issue by Bpazy** [keepassxreboot/keepassxc-browser#573]
Unable to detect form submit when the button is not `<input type="submit">`

## Expected Behavior
Here is an example: https://jsfiddle.net/hk7s2d4x/
‘submit' detected but 'submit2' not detected.

Extension should detect 'submit2' and remember credentials because Chrome can do this.
![1](https://user-images.githubusercontent.com/9838749/60592824-ff654d00-9dd3-11e9-9a29-8c9eb5118ec5.png)

## Current Behavior
'submit2' did not trigger remember action.

## Possible Solution
Maybe we should listen all click events in the form. 

## Steps to Reproduce (for bugs)

https://jsfiddle.net/hk7s2d4x/
1. Input username & password
2. click 'submit' (traditional forms with `input button`). Triggered credential management in Chrome.
3. click `submit2` (event listener which problem happened). Did not triggered credential management in kpx-browser.

## Debug info

KeePassXC - 2.4.3
KeePassXC-Browser - 1.4.6
Operating system: Win
Browser: Chrome

**Comment by droidmonkey** [comment]
Your example doesn't submit the form, it refreshes the page.

**Comment by Bpazy** [comment]
> Your example doesn't submit the form, it refreshes the page.

Just like send request to get cookie and refresh page.

**Comment by droidmonkey** [comment]
I don't understand what you just wrote and it is hard to follow your original post.

**Comment by Bpazy** [comment]
> I don't understand what you just wrote and it is hard to follow your original post.

Issue edited. I mean `Login Button` send `ajax` then `response` contains `Set-Cookie` header. So `location.reload()` makes `Cookie` worked.

**Comment by varjolintu** [comment]
Form submit detection has some changes in PR https://github.com/keepassxreboot/keepassxc-browser/pull/351. Please try if it fixes your issue.

**Comment by droidmonkey** [comment]
Ahhhh I get it now thanks

**Comment by Bpazy** [comment]
fixed now
